sw/device/silicon_creator/rom/e2e/epmp_init_test.c

```diff
@@ -0,0 +1,61 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/csr.h"
+#include "sw/device/lib/dif/dif_lc_ctrl.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/lc_ctrl_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+#include "sw/device/lib/testing/test_framework/status.h"
+#include "sw/device/silicon_creator/lib/epmp_defs.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+
+static bool is_debug_enabled() {
+  dif_lc_ctrl_t lc_ctrl;
+  mmio_region_t lc_reg = mmio_region_from_addr(TOP_EARLGREY_LC_CTRL_BASE_ADDR);
+  CHECK_DIF_OK(dif_lc_ctrl_init(lc_reg, &lc_ctrl));
+  return lc_ctrl_testutils_debug_func_enabled(&lc_ctrl);
+}
```

Comment:
Let's don't trust the test code for this. You can approach this multiple ways but one option could be just print out the config and set the success/fail strings according to the lifecycle state.

Reply:
OK, I opted to inject a preprocessor macro that tells the test whether to expect debugging to be enabled. WDYT?